azure-iot-device/azure/iot/device/common/transport/mqtt/mqtt_topic_helper.py

```diff
@@ -0,0 +1,171 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
lets drop it "helper" from the name though.

mqtt_topic is a good enough module name.